GitUI/CommandsDialogs/FormBrowse.cs

```diff
@@ -1978,6 +1978,7 @@ private void SetGitModule(object sender, GitModuleEventArgs e)
             _gitStatusMonitor.InvalidateGitWorkingDirectoryStatus();
             _submoduleStatusProvider.Init();
             ToolStripFilters.SetBranchFilter(string.Empty, refresh: false);
+            ToolStripFilters.SetRevisionFilter(string.Empty);
```

Comment:
Note for other reviewers: revision filters can be set at FormBrowse startup (as command lind arguments), but SetGitModule() does not execute at startup so filters are not cleared.